src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -42,6 +44,75 @@ static void add_config(
     });
 }
 
+static ss::sstring
+kafka_endpoint_format(const std::vector<model::broker_endpoint>& endpoints) {
+    std::vector<ss::sstring> uris;
+    std::transform(
+      endpoints.cbegin(),
+      endpoints.cend(),
+      std::back_inserter(uris),
+      [](const model::broker_endpoint& ep) {
+          return fmt::format(
+            "{}://{}:{}",
+            (ep.name.empty() ? "plain" : ep.name),
+            ep.address.host(),
+            ep.address.port());
+      });
+    return boost::algorithm::join(uris, ",");
```

Comment:
FYI, I think you can:
```cpp
#include <fmt/ranges.h>
return fmt::format("{}", fmt::join(uris, ", "));
```
Probably the most efficient thing to do is provide a local formatter for the `model::broker_endpoint`, but it's probably overkill.

Reply:
oh, didn't know about fmt::join. will change

Reply:
fmt::join does not seem to like ss::sstring :(

Reply:
It works if you make `uris` a `std::vector<std::string>`.

This make me wonder how often we copy strings from `std::string` to `ss::sstring`.

We'll leave that for another day.